content/docs/configuration/vault.md

```diff
@@ -348,36 +355,60 @@ spec:
           key: token
 ```
 
-<a id="authenticating-with-kubernetes-service-accounts"></a>
+### C. Authenticating with Kubernetes Service Accounts
 
-### Authenticating with Kubernetes Service Accounts
+ℹ️ This documentation works for cert-manager >= v1.12.0.
 
-The [Vault Kubernetes
-Auth](https://developer.hashicorp.com/vault/docs/auth/kubernetes) allows
+The [Vault JWT/OIDC Auth](https://developer.hashicorp.com/vault/docs/auth/jwt/oidc-providers/kubernetes) and the [Vault Kubernetes Auth](https://developer.hashicorp.com/vault/docs/auth/kubernetes) allow
 cert-manager to authenticate to Vault using a Kubernetes Service Account Token
-in order to issue certificates using Vault as a certification authority. The
-Kubernetes service account token can be provided in two ways:
+in order to issue certificates using Vault as a certification authority.
+
+Vault Authentication Method:
+- [Option 1. Use JWT/OIDC Auth](#option-1-vault-authentication-method-use-jwtoidc-auth)
+- [Option 2. Use Kubernetes Auth](#option-2-vault-authentication-method-use-kubernetes-auth)
+
+#### Option 1. Vault Authentication Method: Use JWT/OIDC Auth
```

Comment:
I think JWT can be quite simple too (if your `OIDC discovery endpoint` is publically accessible).

Reply:
(see the 2 bullet points below)

Reply:
yh in the cloud maybe but it is pain in many other environments (esp. Openshift), tokenreview just requires one extra rolebinding in all cases (and making api accessible which is a pre-req of both)

Reply:
I agree, the following section should make that clear:
```
The [JWT/OIDC auth method](https://developer.hashicorp.com/vault/docs/auth/jwt/oidc-providers/kubernetes) should be used instead of the [Kubernetes auth method](#option-2-vault-authentication-method-use-kubernetes-auth) when:
- Your Kubernetes' cluster OIDC discovery endpoint is reachable from the Vault server.
- Your Vault server is not running inside the Kubernetes cluster.
```
(see below)

Reply:
well since 1.15 I don t know if that is true anymore, the only reason we went down the jwt path was the multi-audiences problem.. 

Maybe u want to wait a couple versions and then we can tell people to switch?

Reply:
I added extra comments in this commit: https://github.com/cert-manager/website/pull/1504/commits/3697c70c5eae3ed25526b04e7cd8fbb8ffbd48e0